cloud: add steps for creating private endpoints on Google Cloud and Azure (#22800)
pingcap/docs: tidb-cloud/set-up-private-endpoint-connections-on-azure.md

    >
    > For each TiDB Cloud Dedicated cluster, the corresponding endpoint service is automatically created 3 to 4 minutes after the cluster creation.

> 2. Create the private endpoint by using either Azure Portal or Azure CLI.

Reviewer: Use the official product style `Azure portal`.

```suggestion
2. Create the private endpoint by using either the Azure portal or Azure CLI.
```

<SimpleTab>

> <div label="Use Azure Portal">

Reviewer: For consistency with `Azure portal` in the steps, use lowercase `portal` in the tab label as well.

```suggestion
<div label="Use Azure portal">
```

6. Continue clicking **Next** to go through the remaining configuration tabs and complete the required settings. Then, click **Create** to create and deploy the private endpoint. It might take a few seconds for Azure to complete the deployment. For more information, see [Create a private endpoint](https://learn.microsoft.com/en-us/azure/private-link/create-private-endpoint-portal?tabs=dynamic-ip#create-a-private-endpoint) in Azure documentation.
7. After the private endpoint is created and deployed, click **Go to resource**, and then do the following:

    - Click **Settings** > **Properties** in the left navigation pane, and copy its **Resource ID** for later use.

        ![Azure private endpoint resource ID](/media/tidb-cloud/azure-private-endpoint-resource-id.png)

    - Click **Settings** > **DNS configuration** in the left navigation pane, and then copy its **IP address** for later use.

        ![Azure private endpoint DNS IP](/media/tidb-cloud/azure-private-endpoint-dns-ip.png)

</div>
<div label="Use Azure CLI">

1. Sign in to Azure CLI and select your subscription:

    ```bash
    az login
    az account set --subscription ${your_subscription_id}
    ```

> 2. Create the private endpoint by using the TiDB Cloud resource ID copied in Step 2:

Reviewer: reason: `Step 2` is the current section, so `copied in Step 2` can read as self-referential. 

```suggestion
2. Create the private endpoint by using the TiDB Cloud resource ID that you copied from the **Create Azure Private Endpoint Connection** dialog:
```

Reviewer: <img width="1114" height="240" alt="image" src="https://github.com/user-attachments/assets/6aed5177-a686-46f9-ac3b-5531fbb4b7c2" />
